src/index.js

```diff
@@ -0,0 +1,265 @@
+import { setDate } from './modules/helpers';
+// eslint-disable-next-line
+import {
+  Project, projectsToDom, deleteProject, checkProject,
+} from './modules/projects';
+// eslint-disable-next-line
```

Comment:
Kindly avoid the ESLInt inline disabling rule, it is the same as altering the ESLint config files